demos/object_detection_demo/main.cpp

```diff
@@ -298,11 +295,11 @@ int main(int argc, char *argv[]) {
         slog::info << presenter.reportMeans() << slog::endl;
     }
     catch (const std::exception& error) {
-        slog::err << "[ ERROR ] " << error.what() << slog::endl;
+        std::cerr << "[ ERROR ] " << error.what() << '\n';
```

Comment:
There is no reason to use `slog::err` if `[ ERROR ] ` is added manually.

Reply:
It's better remove slog. There's one bit merit of slog - if it would be required we can easily froward it to another pipe (without OS means) - i.e. to the file, or pass through filters, etc. For instance, in test system. So it's better not mix 2 approaches - either everyhting in app should go through cerr, or through slog (I'd prefer the latter one).

Reply:
Removed